cr-db/cr/db/loader.py

```diff
@@ -25,29 +27,53 @@ def load_data(filename, settings=None, clear=None):
             collection.insert(obj)
 
 
-def load_dataset(csv_filename, db):
-    with file(csv_filename, 'rU') as csv_file:
-        csv_data = csv.reader(csv_file)
-        headers = csv_data.next()
+def load_bulk_data(filename, settings=None, clear=None):
+    if settings is None:
+        settings = global_settings
+        global_settings.update(json.load(file(sys.argv[1])))
+
+    db = connect(settings)
+
+    obj_name = os.path.basename(filename).split('.')[0]
+
+    collection = getattr(db, obj_name)
+
```

Comment:
so, one collection per dataset?  Any possible long-term scaling issues with this?  How would you handle name clashing for instance?

Reply:
> so, one collection per dataset? 
maybe could be useful to use one collection for the users and do not use a new collection for new coming users datasets. 

> Any possible long-term scaling issues with this? 
can you be more specific? I'm afraid I didn't understand the question :(

> How would you handle name clashing for instance?
using the same collection for users maybe we shouldn't have name clashing for collections, in case we can use a uuid instead of the name of the file